utils/guardrails/prompt.yaml

```diff
@@ -0,0 +1,28 @@
+_type: prompt
+input_types: {}
+input_variables:
+- guardrails
+- conversation
+- role
+name: null
+output_parser: null
+partial_variables: {}
+template: |
+  <|begin_of_text|>[INST] Task: Check if there is unsafe content in '{role}' messages in conversations according our safety policy with the below categories.
+
+  <BEGIN UNSAFE CONTENT CATEGORIES>
+  {guardrails}
+  <END UNSAFE CONTENT CATEGORIES>
+
+  <BEGIN CONVERSATION>
+
+  {conversation}
+
+  <END CONVERSATION>
+
+  Provide your safety assessment for ONLY THE LAST {role} in the above conversation:
+  - First line must read 'safe' or 'unsafe'.
+  - If unsafe, a second line must include a comma-separated list of violated categories. [/INST]
```

Comment:
Could we use `langchain` + the `pydantic` output parser in the future to better structure this answer?

Reply:
would be possible, but since the llamaguard model was finetuned generating this specific output format with this prompt, I don't think it is necessary